recipe/meta.yaml

```diff
@@ -7,15 +7,13 @@ package:
 source:
   fn: libevent-{{ version }}.tar.gz
   url: https://github.com/libevent/libevent/archive/release-{{ version }}-stable.tar.gz
-  sha256: 52c9db0bc5b148f146192aa517db0762b2a5b3060ccc63b2c470982ec72b9a79
-  patches:
-    - 0374b55942e533a3c3997439481a8d05d6c8f729.patch
+  sha256: 229393ab2bf0dc94694f21836846b424f3532585bac3468738b7bf752c03901e
 
 build:
   number: 0
   run_exports:
     # https://abi-laboratory.pro/tracker/timeline/libevent/
-    - {{ pin_subpackage('libevent', max_pin='x.x') }}
+    - {{ pin_subpackage('libevent', max_pin='x.x.x') }}
```

Comment:
I've tightened the `run_exports`, but this doesn't help users who have things built against `2.1.0` as they could be installed with `2.1.1` and be broken. How should we fix that case?

Reply:
can we set 2.1.0 to broken and rebuild with stricter pinning?

Reply:
We could do that. Though that will result in any packages that require `libevent` version `2.1.0` to fail to install.

Reply:
We could fix the `max_pin` before this goes in. Then `2.1.0` would be correct.

Reply:
That would help any new packages that are built using `2.1.0`, but what would do for the existing ones? Should we hot-fix?

Reply:
hot fix would work too